tilelang/tools/lower_trace/diff.py

```
@@ -0,0 +1,411 @@
+            for k in range(min(context, i2 - i1)):
+                before_collapse[i1 + k] = False
+                after_collapse[j1 + k] = False
+            for k in range(min(context, i2 - i1)):
+                before_collapse[i2 - 1 - k] = False
+                after_collapse[j2 - 1 - k] = False
+
+    rows = []
+
+    for tag, i1, i2, j1, j2 in opcodes:
+        if tag == "equal":
+            for i, j in zip(range(i1, i2), range(j1, j2)):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)